.coveragerc

```diff
@@ -918,6 +918,10 @@ omit =
     homeassistant/components/systemmonitor/sensor.py
     homeassistant/components/tado/*
     homeassistant/components/tado/device_tracker.py
+    homeassistant/components/tahoma/__init__.py
+    homeassistant/components/tahoma/binary_sensor.py
+    homeassistant/components/tahoma/coordinator.py
+    homeassistant/components/tahoma/tahoma_entity.py
```

Comment:
Exclude the `overkiz_executor.py` module too.

Reply:
Fixed.